lib/App.ts

```diff
@@ -1,25 +1,34 @@
-import Component from "./Component.js";
+import _ from "lodash";
 import Navigo from "navigo";
+import AppOptions from "./AppOptions.js";
+import Component from "./Component.js";
 
 /**
  * This class creates a page component for each route in the application and initializes it and all its children.
  */
 export default class App {
+    private isAuthed = false; // whether the base page was loaded with ?isAuthed
```

Comment:
No need to shorten words unnecessarily here - `isAuthorized` would be more readable, actually a real word, and will be removed from the URL regardless

Reply:
Good catch, refactoring that.